config/locales/views/visits.fr.yml

```diff
@@ -17,6 +17,7 @@ fr:
       person_you_met: Personne que vous avez rencontrée
       diagnosis: Analyse des besoins de l'entreprise
       cta_diagnosis: Faire l'analyse des besoins
+      diagnosis_date: "Diagnostic datant du : %{date}"
```

Comment:
Columns means "deux points". I suggest you change this line for one of those :

>Analyse datant du %{date}

Or :

>Date de l'analyse : %{date}